signature/policy_reference_match_test.go

```diff
@@ -136,7 +136,7 @@ var prmExactMatchTestTable = []prmSymmetricTableTest{
 	{"busybox" + digestSuffix, "busybox" + digestSuffix, true}, // NOTE: This is not documented; signing digests is not recommended at this time.
 	// Non-canonical reference format is canonicalized
 	{"library/busybox:latest", "busybox:latest", true},
-	{"docker.io/library/busybox:latest", "busybox:latest", true},
+	{"<local>/library/busybox:latest", "busybox:latest", true},
```

Comment:
This redefines the meaning of existing signatures, e.g. https://rhelblog.redhat.com/2016/07/22/container-image-signing/ shows a reference that does not contain a host name.

(_Recent_ code uses the fully explicit format in created signatures, I’m afraid that hasn’t been originally the case.)